odxtools/compumethods/readcompumethod.py

```diff
@@ -48,8 +48,14 @@ def _parse_compu_scale_to_linear_compu_method(scale_element,
     offset = computation_python_type(next(nums).text)
     factor = computation_python_type(next(nums).text)
     if coeffs.find("COMPU-DENOMINATOR/V") is not None:
-        kwargs["denominator"] = int(
-            coeffs.find("COMPU-DENOMINATOR/V").text)
+        try:
+            kwargs["denominator"] = int(
+                coeffs.find("COMPU-DENOMINATOR/V").text)
+        except ValueError as ex:
+            pass
+        finally:
```

Comment:
the float parsing needs to be in the catch block, not in the finally
```suggestion
```

Reply:
I'd like to propose something like this:
```python
        denom_elem = coeffs.find("COMPU-DENOMINATOR/V")
        kwargs["denominator"] = int(denom_elem.text) if denom_elem is not None else 1
```

Reply:
the point of the pr is to address the fact that the value could be a float instead of an int, not sure how the default 1 value helps with that.